### ISSUE TITLE
accessibility: Make sure the survey can be filled without scripting enabled!!

### ISSUE THREAD

**Issue by tahini** [chairemobilite/evolution#36]
That would be a big piece, but from https://webaim.org/techniques/formvalidation/ it says

```
The form must also submit to the server if client-side scripting is unavailable. Forms that rely solely on script functions or event handlers should be avoided. Instead, use a true URL action value for the form. Client-side validation can still be invoked, and it would be processed first if scripting is enabled.
```

**Comment by samuel-duhaime** [comment]
I agree with @kaligrafy, but when I tried to do a survey with JavaScript disabled in my browser, I only got a blank white page. We should leave a comment informing users that they need to enable JavaScript to use this website.
